src/headless/plugins/chat/model.js

```diff
@@ -228,6 +228,7 @@ const ChatBox = ModelWithContact.extend({
                 !this.handleChatMarker(attrs) &&
                 !(await this.handleRetraction(attrs))
         ) {
+            const message_retraction = await api.hook('onMessage', this, { retract: false });
```

Comment:
@jcbrand so this hook verifies the fact that we don't need to create a new message, right?